lib/FrontendTool/FrontendTool.cpp

```diff
@@ -184,20 +185,39 @@ static bool emitMakeDependenciesIfNeeded(DiagnosticEngine &diags,
 }
 
 namespace {
+struct SwiftModuleTraceInfo {
+  std::string ModulePath;
```

Comment:
Y'know, if we're making a new format, should we also print the module name? It might be useful for us.

Reply:
Yes, that sounds useful enough to be included.